pkg/skaffold/build/tag/git_commit.go

```diff
@@ -142,6 +151,25 @@ func runGit(workingDir string, arg ...string) (string, error) {
 	return strings.TrimSpace(string(out)), nil
 }
 
+func runGitLines(workingDir string, arg ...string) ([]string, error) {
+	out, err := runGit(workingDir, arg...)
+	if err != nil {
+		return nil, err
+	}
+
+	var lines []string
+
+	scanner := bufio.NewScanner(strings.NewReader(out))
+	for scanner.Scan() {
+		line := strings.TrimSpace(scanner.Text())
+		if line != "" {
+			lines = append(lines, line)
+		}
+	}
+
+	return lines, nil
+}
+
 func commitOrTag(currentTag string, tags []string, opts *Options) string {
```

Comment:
why does this need a slice of tags?

Can git describe --tags --always return more than one tag?

```shell
$ git tag test
$ git tag test2
$ git describe --tags --always
test
```

Reply:
Go-git returns a slice. Don’t know why